.phpstan-baseline.php

```diff
@@ -7186,7 +7186,7 @@
 $ignoreErrors[] = [
 	'message' => '#^Property Profile\\:\\:\\$profileRight \\(array\\) does not accept null\\.$#',
 	'identifier' => 'assign.propertyType',
-	'count' => 2,
+	'count' => 1,
```

Comment:
There should be ne new entries in phpstan baseline. Either issues have to be fixed; or properly ignored in code for false positives.

Reply:
Sorry, I misread